src_tests/unittests/osm_configurator/view/states/test_positioned_frame.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+import src.osm_configurator.view.states.positioned_frame as positioned_frame_i
+import src.osm_configurator.view.toplevelframes.top_level_frame as top_level_frame_i
+import src.osm_configurator.view.states.state_manager as state_manager_i
+
+
+def test_type_errors():
```

Comment:
Please add comments for better readability

Reply:
all Test have revieved comments